test_search.py

```diff
@@ -1,11 +1,9 @@
 import time
 import pytest
-
```

Comment:
These spaces should still be here. See, https://www.python.org/dev/peps/pep-0008/#imports